src/sentry/utils/samples.py

```diff
@@ -81,6 +90,19 @@ def load_data(platform, default=None):
         "method": "GET"
     }
 
+    now = datetime.utcnow()
+
+    # Make breadcrumb timestamps relative to right now so they make sense
+    breadcrumbs = data.get('sentry.interfaces.Breadcrumbs')
+    if breadcrumbs is not None:
+        duration = 1000
+        values = breadcrumbs['values']
+        for value in reversed(values):
+            value['timestamp'] = milliseconds_ago(now, duration)
```

Comment:
Unless that is considered strictly superior, I'd rather keep as-is rather than re-open, try this change, test, etc. It seems like a wash to me for such a small thing (plus `milliseconds_ago` is pretty clear about what's going on).